CHANGELOG.md

```diff
@@ -5,6 +5,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v3.41.0](https://github.com/buildkite/agent/tree/v3.42.0) (2023-01-05)
+[Full Changelog](https://github.com/buildkite/agent/compare/v3.41.0...v3.42.0)
+
+ ### Added
+ - Add `git config protocol.file.allow=always` to tests [#1895](https://github.com/buildkite/agent/pull/1895) (@triarius)
+ - Add an in-built hierarchical status page [#1873](https://github.com/buildkite/agent/pull/1873) (@DrJosh9000)
+ - Add an `agent-startup` hook that fires at the same time as the `agent-shutdown` hook is registered [#1778](https://github.com/buildkite/agent/pull/1778) (@donalmacc)
+ - Add `test-collector` plugin for internal metric collection [#1881](https://github.com/buildkite/agent/pull/1881) (@DrJosh9000)
```

Comment:
"internal metric collection" sounds wrong? But the PR doesn't add anything for users, only us, so might not be worth mentioning.

Reply:
Removed from changelog :)